benchmark/cluster/vgg16/vgg16_fluid.py

```diff
@@ -196,14 +198,28 @@ def train_loop(exe, trainer_prog):
                     feed={"pixel": img_data,
                           "label": y_data},
                     fetch_list=[avg_cost, batch_acc, batch_size])
+                return loss, acc, b_size
+
+            if args.profile and args.task_index == 0:
```

Comment:
maybe here we can use trainer_id == 0

Reply:
I think task_index and trainer_id mean the same thing here? vgg_fluid.py only has task_index, but no trainer_id.